packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -4,28 +4,55 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { Aliases, AuthInfo } from '@salesforce/core';
-import { ForceConfigGet } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
-
+import {
+  Aliases,
+  AuthInfo,
+  ConfigAggregator,
+  ConfigFile
+} from '@salesforce/core';
+import * as path from 'path';
+import { getRootWorkspacePath, hasRootWorkspace } from './index';
 export class OrgAuthInfo {
-  public static async getDefaultUsernameOrAlias(
-    vscodePath: string
-  ): Promise<string | undefined> {
-    const forceConfig = await new ForceConfigGet().getConfig(
-      vscodePath,
-      'defaultusername'
-    );
-    return forceConfig.get('defaultusername');
+  public static async getDefaultUsernameOrAlias(): Promise<string | undefined> {
+    try {
```

Comment:
If we can't find a local `sfdx-config.json` or there is no `defaultusername`/`defaultdevhubusername` set, we default to calling the global values. If the global default is not found either, I thought it would make sense to log an error explaining that either the `defaultdevhubusername` or `defaultusername` could not be found. If this behavior makes sense, @ruthemmanuelle what would you recommend the message to display?

Reply:
@AnanyaJha and I talked about this via Slack. My suggested language: `No default {org|Dev Hub} is set. Run {"SFDX: Create a Default Scratch Org" or "SFDX: Authorize an Org"|"SFDX: Authorize a Dev Hub"} to set one.`